src/SDKs/Billing/Management.Billing/Microsoft.Azure.Management.Billing.csproj

```diff
@@ -5,7 +5,7 @@
 
 	<PropertyGroup>
 		<Description>Microsoft Azure Billing Management Library</Description>
-		<Version>2.0.0-preview</Version>
+		<Version>2.1.0-preview</Version>
```

Comment:
We highly recommend adding `PackageReleaseNotes`

Reply:
Done